src/jit/morph.cpp

```diff
@@ -2442,6 +2442,71 @@ void fgArgInfo::EvalArgsToTemps()
 #endif
 }
 
+#if defined(UNIX_X86_ABI) && FEATURE_FIXED_OUT_ARGS
+// Reverse slot number for each arguments so that arguments will be placed
+// in the same order when changed from 'push' to 'move' instruction.
+// It would be better not to do this if possible.
+void fgArgInfo::ReverseArgumentSlot()
+{
+    // To get the padding amount, sum up all the slots and get the remainder for padding
+    unsigned curInx;
+    unsigned numSlots     = 0;
+    unsigned numStackArgs = 0;
+
+    for (curInx = 0; curInx < argCount; curInx++)
+    {
+        fgArgTabEntryPtr curArgTabEntry = argTable[curInx];
+        if (curArgTabEntry->numSlots > 0)
+        {
+            // The argument may be REG_STK or constant or register that goes to stack
+            assert(nextSlotNum >= curArgTabEntry->slotNum);
+
+            numSlots += curArgTabEntry->numSlots;
```

Comment:
Thanks, it's from copy of previous stack alignment. Need to remove this and some comments also.